### DISCUSSION TITLE
Need Help

### DISCUSSION THREAD

**Discussion by Mansaylon** [GameServerManagers/LinuxGSM#3234] [Q&A] [answered]
Hello

I have been trying to create a dedicated Valheim server for days.
The server is based on Centos 8.
SteamCMD is installed.

The Valheim server data from Steam are loaded in the standard directory. Unfortunately, further installations in other directories did not work.

It is also confusing that the server ID on Steam should be 896660 and 892970.

Looking further for a solution, I found LinuxGSM_.

My english is humble. Do I understand correctly, LinuxGSM_ is an additional tool that is used to SteamCMD?

Desperately looking for help.

Bye
Mansaylon

**Comment by dgibbs64** [comment]
Hello @Mansaylon please check out the LinuxGSM website https://linuxgsm.com for full details of what LinuxGSM does. If you are struggling with understanding English see if a translation website will help. Thank you

**Comment by Mansaylon** [comment, marked as answer]
Hi


Thank you .... but my English is not good enough.
And the translators usually do not provide a usable result (when
things get complex).
I was finally able to solve everything yesterday. Linux server runs
with steamCMD. 2 Valheim worlds run parallel and I was able to push a
private game onto the dedicated server. ;-)


Best Regards
Reto




Am Samstag, den 13.02.2021 um 12:12 schrieb Daniel Gibbs:








Hello @Mansaylon [1] please check out the LinuxGSM website
https://linuxgsm.com for full details of what LinuxGSM does. If you
are struggling with understanding English see if a translation website
will help. Thank you



—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub [2], or unsubscribe
[3].

[ { "@context": "http://schema.org", "@type": "EmailMessage",
"potentialAction": { "@type": "ViewAction", "target":
"https://github.com/GameServerManagers/LinuxGSM/discussions/3234#discussioncomment-365047",
"url":
"https://github.com/GameServerManagers/LinuxGSM/discussions/3234#discussioncomment-365047",
"name": "View Discussion" }, "description": "View this Discussion on
GitHub", "publisher": { "@type": "Organization", "name": "GitHub",
"url": "https://github.com" } } ]



Links:
------
[1] https://github.com/Mansaylon
[2]
https://github.com/GameServerManagers/LinuxGSM/discussions/3234#discussioncomment-365047
[3]
